oss/libpopcnt/libpopcnt.h

```diff
@@ -95,7 +95,7 @@
   #define HAVE_AVX512
 #endif
 
-#if defined(X86_OR_X64) && !defined(_M_ARM64EC)
```

Comment:
We should have documented this change, but we added ARM64EC because the Windows build system breaks otherwise.

Reply:
so i need change this to ARM64EC?

Reply:
this needs to get changed back, yea